src/client.ts

```diff
@@ -115,6 +123,29 @@ export class FarosClient {
     }
   }
 
+  async addModels(
+    graph: string,
+    models: string,
+    schema: string
+  ): Promise<void> {
+    if (this.graphVersion !== GraphVersion.V1) {
+      throw new VError(
+        `Adding models is not supported for ${this.graphVersion} graphs`
+      );
+    }
+    try {
+      await this.api.post(`/graphs/${graph}/models?schema=${schema}`, models, {
```

Comment:
since `schema` an optional argument, we should move `?schema=${schema}` to `params` as optional